File: JavaBasic/94_grade_switch.java
// 94	94_grade_switch.java	Display grade (A/B/C/D/F) using switch case on marks range.
public class _94_grade_switch {
    public static void main(String[] args) {
        int marks = 82;
        System.out.println("Marks: " + marks);
        if (marks >= 90) System.out.println("Grade: A");
        else if (marks >= 75) System.out.println("Grade: B");
        else if (marks >= 60) System.out.println("Grade: C");
        else if (marks >= 40) System.out.println("Grade: D");
        else System.out.println("Grade: F");
    }
}

File: JSBASIC/94_grade_switch.js
// 94	94_grade_switch.js	Display grade (A/B/C/D/F) using switch case on marks range.

let marks = 82;
console.log("Marks:", marks);
let grade = Math.floor(marks / 10);
switch (true) {
    case (marks >= 90): console.log("Grade: A"); break;
    case (marks >= 75): console.log("Grade: B"); break;
    case (marks >= 60): console.log("Grade: C"); break;
    case (marks >= 40): console.log("Grade: D"); break;
    default: console.log("Grade: F");
}
